tests/test_pvc_concurrent_deletion_creation.py

```diff
@@ -0,0 +1,215 @@
+"""
+Test to verify concurrent creation and deletion of multiple PVCs
+"""
+import subprocess
+import logging
+import pytest
+
+from ocs import defaults, constants, ocp, exceptions
+from utility.utils import run_cmd
+from ocsci.testlib import tier1, ManageTest
+from resources.pod import get_ceph_tools_pod
+from resources.pvc import PVC
+from tests.fixtures import (
+    create_rbd_storageclass, create_ceph_block_pool, create_rbd_secret
+)
+
+log = logging.getLogger(__name__)
+PVC_OBJS = []
+TEST_PROJECT = 'test-project'
+PROJECT = ocp.OCP(kind='Project', namespace=TEST_PROJECT)
+
+
+@pytest.fixture()
+def test_fixture(request):
+    """
+    Setup and teardown
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+    request.addfinalizer(finalizer)
+    setup(self)
+
+
+def setup(self):
+    """
+    Create new project
+    Create PVCs
+    """
+    # Create new project
+    assert PROJECT.new_project(TEST_PROJECT), (
+        f'Failed to create new project {TEST_PROJECT}'
+    )
+
+    # Create 100 PVCs
+    create_multiple_pvc(
+        self.pvc_base_name, self.number_of_pvc, self.sc_obj.name
+    )
+    log.info(f'Created initial {self.number_of_pvc} PVCs')
+    self.pvc_objs_initial = PVC_OBJS[:]
+    PVC_OBJS.clear()
+
+    # Verify PVCs are Bound and fetch PV names
+    for pvc in self.pvc_objs_initial:
+        pvc.reload()
+        assert pvc.status == constants.STATUS_BOUND, (
+            f'PVC {pvc.name} is not Bound'
+        )
+        self.initial_pvs.append(pvc.backed_pv)
+    log.info(f'Initial {self.number_of_pvc} PVCs are in Bound state')
+
+
+def teardown(self):
+    """
+    Delete PVCs
+    Delete project
+    """
+    # Delete newly created PVCs
+    command = (
```

Comment:
create a function for async_delete_pvcs() under pvc.py as helper function

Reply:
To delete PVCs in async way, run_async method in utils.py can be used. I think the same approach is appropriate in this test case.

Reply:
we have PVC class, I think the functionality of execute command can be moved inside that class if it doesn't already exists there.

Reply:
will take this up as follow on PR